src/pybind/mgr/dashboard/frontend/src/app/core/auth/login/login.component.ts

```diff
@@ -63,7 +65,11 @@ export class LoginComponent implements OnInit {
 
   login() {
     this.authService.login(this.model).subscribe(() => {
-      const url = _.get(this.route.snapshot.queryParams, 'returnUrl', '/');
+      const urlPath = this.postInstalled ? '/' : '/create-cluster';
+      let url = _.get(this.route.snapshot.queryParams, 'returnUrl', urlPath);
+      if (!this.postInstalled && this.route.snapshot.queryParams['returnUrl'] === '/dashboard') {
+        url = '/create-cluster';
+      }
```

Comment:
What's this for?

Reply:
When you try to login using an empty state link, for eg: https://192.168.122.17:8443/, it will automatically add the returnUrl as /dashboard and tries to redirect to dashboard page.

Reply:
Understood. I was wondering if for the sake of simplicity we should (temporarily) remove all the auto-guessing (cluster status, post-login redirection) and simply unconditionally display the wizard in the cluster-> submenu...

Reply:
Since we came this far, I'd say we keep it as it is. Otherwise I guess we may have to rediscuss the workflow again and make a lot of changes to this.